frontend/src/components/events/organisms/event_section.jsx

```diff
@@ -17,6 +17,18 @@ const EventSection = ({ events, isUpcoming }) => {
           <Text text={sectionTitle} type="heading" />
         </div>
         {events.map(({ name, start, logo, url, id, description }) => {
+          const maxChar = 200;
+          if (description.text.length > maxChar) {
```

Comment:
this can be written a bit more succinctly

```
description.text = description.text.length > 200 ? description.text.substring(0, 200) + '...' : description.text
```